packages/cli-e2e/__tests__/angular.specs.ts

```diff
@@ -137,37 +136,33 @@ describe('ui:create:angular', () => {
 
   const startApplication = async (
     processManager: ProcessManager,
-    debugName = 'angular-server',
-    errorCallback: (error: string) => void = () => {}
+    debugName = 'angular-server'
   ) => {
+    const args = ['npm', 'run', 'start'];
+    if (process.platform === 'win32') {
+      args[0] = npmJsPath();
+      args.unshift('node');
+    }
+
     const serverTerminal = new Terminal(
-      'npm',
-      ['run', 'start'],
+      args.shift()!,
```

Comment:
I'm seeing that this is repeated for each invocation of the `Terminal` constructor.

Could that be handled internally by the class ? So that we don't have to "remember" to do it from a user POV.

Reply:
Alas no, because the arg shifting is only for `npm` related task.